.github/workflows/tests.yml

```diff
@@ -48,7 +48,10 @@ jobs:
       - name: print runner CPU
         run: lscpu
 
-      - run: ./ci_scripts/all_tests.sh
+      - name: check if roc files are properly formatted
+        run: ROC=./roc_nightly/roc ./ci_scripts/check_format.sh
```

Comment:
I took the opportunity to use `ROC=... script.sh` like we have for basic-cli, it's convenient to have.